src/utils.js

```diff
@@ -165,3 +165,17 @@ export function ensureDefinedConfig(object, requester) {
     }
   });
 }
+
+/**
+ * This function helps to parse api query params
+ *
+ * @param {String} mainURI
+ * @param {Array} queryParams Array with object with strings to set query name and value
+ *  {name: '', value, ''}
+ */
+
+export function parseUrlQueryParams(mainURI, queryParams = []) {
```

Comment:
I'm sensitive to us adding more utility functions here.  If we add something to the utilities here, it should be something we expect MFEs will need; these are intended to be common operations that MFEs will generally all do.  Since `parseUrlQueryParams` is just used internally, I think I'd almost rather we just inline it where we need it:

```
const params = new URLSearchParams();
params.append('mfe', <mfe name>);
const url = `${MFE_CONFIG_API_URL}?${params.toString()}`;
```

Having those three lines up in the `runtimeConfig` function feels pretty clear and simple to understand.